scipy/conftest.py

```diff
@@ -180,22 +180,69 @@ def check_fpu_mode(request):
               'that are not valid input when `SCIPY_ARRAY_API` is used.'))
 
 
+def _backends_kwargs_from_request(request, skip_or_xfail):
+    """A helper for {skip,xfail}_xp_backends"""
+    # do not allow multiple backends
+    args_ = request.keywords[f'{skip_or_xfail}_xp_backends'].args
+    if len(args_) > 1:
+        # np_only / cpu_only has args=(), otherwise it's ('numpy',)
+        # and we do not allow ('numpy', 'cupy')
+        raise ValueError(f"multiple backends: {args_}")
+
+    markers = list(request.node.iter_markers(f'{skip_or_xfail}_xp_backends'))
+    backends = []
+    kwargs = {}
+    for marker in markers:
+        if marker.kwargs.get('np_only'):
+            kwargs['np_only'] = True
+            kwargs['exceptions'] = marker.kwargs.get('exceptions', [])
+        elif marker.kwargs.get('cpu_only'):
+            if not kwargs.get('np_only'):
+                # if np_only is given, it is certainly cpu only
+                kwargs['cpu_only'] = True
+                kwargs['exceptions'] = marker.kwargs.get('exceptions', [])
+
+        # add backends, if any
+        if len(marker.args) > 0:
+            backend = marker.args[0]  # was a tuple, ('numpy',) etc
+            backends.append(backend)
+            kwargs.update(**{backend: marker.kwargs})
+
+    return backends, kwargs
+
+
 @pytest.fixture
 def skip_xp_backends(xp, request):
-    """See the `skip_or_xfail_xp_backends` docstring."""
+    """skip_xp_backends(backend, reason, np_only=False, cpu_only=False, exceptions=None)
+
+    Skip a decorated test for the provided backend.
+
+    See `skip_or_xfail_backends` docstring for details. Note that, contrary to
+    `skip_or_xfail_backends`, the ``backend`` argument here is a single string: this
+    function only skips a single backend at a time. To skip multiple backends,
+    provide multiple decorators.
```

Comment:
Could you clarify that it is also fine to not provide a backend (use e.g. `cpu_only=True` instead).